Models/Professors.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, String, Integer
+
+Base = declarative_base()
+
+class professor(Base):
+    __tablename__ = 'professors'
+    nugget = Column(String)
+    first_name = Column(String)
+    middle_name = Column(String)
+    last_name = Column(String)
```

Comment:
Why do we need to store these separately? Why not just store the professor's name as a single string?

See http://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/